Adapt bug_size to new max_glob_str variable in texmf.cnf
---
 texk/bibtex8/bibtex.h |    2 +-
 texk/bibtexu/bibtex.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: texlive-bin-2011.20120408/texk/bibtex8/bibtex.h
===================================================================
--- texlive-bin-2011.20120408.orig/texk/bibtex8/bibtex.h	2010-05-03 00:24:45.000000000 +0900
+++ texlive-bin-2011.20120408/texk/bibtex8/bibtex.h	2012-04-09 19:16:52.452321697 +0900
@@ -267,7 +267,7 @@
 #define AUX_STACK_SIZE              20
 #define MAX_BIB_FILES               20
 
-#define BUF_SIZE                   10000
+#define BUF_SIZE                   20000
 #define MAX_CITES                  750
 #define MAX_FIELDS                 5000
 #define MAX_STRINGS                4000
Index: texlive-bin-2011.20120408/texk/bibtexu/bibtex.h
===================================================================
--- texlive-bin-2011.20120408.orig/texk/bibtexu/bibtex.h	2010-05-03 00:30:24.000000000 +0900
+++ texlive-bin-2011.20120408/texk/bibtexu/bibtex.h	2012-04-09 19:16:52.452321697 +0900
@@ -267,7 +267,7 @@
 #define AUX_STACK_SIZE              20
 #define MAX_BIB_FILES               20
 
-#define BUF_SIZE                   10000
+#define BUF_SIZE                   20000
 #define MAX_CITES                  750
 #define MAX_FIELDS                 5000
 #define MAX_STRINGS                4000
